project-base/assets/js/bin/trans.js

```diff
@@ -1,8 +1,18 @@
 #!/usr/bin/env node
 
 const processTrans = require('../commands/translations/process');
-const dirWithJsFiles = './assets/js/';
-const dirWithTranslations = './translations/';
-const outputDirForExportedTranslations = dirWithJsFiles;
+const dirWithTranslations = './translations/*.po';
+const outputDirForExportedTranslations = './assets/js/';
```

Comment:
current dir (./) is the root of the project? Is it somehow dependent on where I run the command? (is it even possible?)

Reply:
You have to run command in same path where is package.json. Without package.json the command (and other commands) don't work.